index.html

```diff
@@ -4,9 +4,645 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/2d6ecfbd70.js" crossorigin="anonymous"></script>
     <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
 </head>
 <body>
+    <!-- Header Section -->
+    <header class="header flex">
+        <!-- Navbar -->
+        <nav class="navbar flex">
+            <div class="left-nav flex">
+                <i class="fas fa-bars"></i>
+                <i class="fas fa-search"></i>
+            </div>
+            <div class="middle-nav">
+                <img src="./img/logo.png" alt="logo">
+            </div>
```

Comment:
This logo needs to be a link 
![image](https://user-images.githubusercontent.com/61479648/87701573-98140400-c76e-11ea-83dd-6b34410e84ac.png)